internal/app/connectconformance/testsuites/cancellation.yaml

```diff
@@ -0,0 +1,100 @@
+name: Cancellation
+testCases:
+# TODO - The server stream tests are failing with grpc due to issues with
```

Comment:
As we discussed offline, this is because of the `stream.Header` call in the gRPC client implementation. That call blocks until header metadata is sent, but the server stream endpoint sends the header information with the first response. So, the `stream.Header` call blocks until a response is sent and _then_ the context is cancelled. So right now, the expectations for responses on server streams won't work.

This can be fixed with a change on the reference server and the gRPC server:

Reference Server
- should call `stream.Send(nil)` after setting the headers.

gRPC Server
- should call `stream.SendHeader(headerMD)` (instead of `SetHeader`)

However, doing the above causes a "protocol error: incomplete envelope: context deadline exceeded" error to be thrown by Connect-Go on the server stream timeout tests.